src/diff_tform.c

```diff
@@ -296,6 +296,9 @@ static int normalize_find_opts(
 	if (opts->flags & GIT_DIFF_BREAK_REWRITES)
 		opts->flags |= GIT_DIFF_FIND_REWRITES;
 
+	if (opts->flags & GIT_DIFF_FIND_NO_RENAMES)
+		opts->flags &= ~GIT_DIFF_FIND_ALL;
```

Comment:
Oops, upon looking at this again, I realize that `DIFF_FIND_ALL` doesn't involve you unsetting your own bit.  So disregard that complaint, obviously.